sidm/tools/ffschema.py

```diff
@@ -187,7 +187,8 @@ def _build_collections(self, branch_forms):
             # handle non-vector objects
             if obj not in branch_forms:
                 branch_forms[obj] = zip_forms(
-                    {a: branch_forms.pop(f"{obj}_{a}") for a in attributes if a != "n"},
+                    #{a: branch_forms.pop(f"{obj}_{a}") for a in attributes if a != "n"}, #Original line ------------------------------ Changed here
```

Comment:
Hi @yfv2ev, I think you can remove this commented line (your fix is probably good to keep from now on)

Reply:
Fixed